plumbing/transport/ssh/common.go

```diff
@@ -11,7 +11,16 @@ import (
 )
 
 // DefaultClient is the default SSH client.
-var DefaultClient = common.NewClient(&runner{})
+var DefaultClient = NewClient()
+
+// NewClient creates a new SSH client with the given options.
+func NewClient(opts ...ClientOption) transport.Transport {
```

Comment:
But you cannot use it to have a _global wrapper_ (instead of a _global override_). As in:

```
if c.HostKeyCallback != nil {
    c.HostKeyCallback = globalWrapper(c.HostKeyCallback)
}
```

Note that we discussed about this offline and agreed to make the API simpler with just a default `ssh.ClientConfig` that is copied. I will update the PR accordingly.